nucleus/pom.xml

```diff
@@ -160,7 +160,7 @@
         <mimepull.version>1.9.12</mimepull.version>
 
         <!-- Apache Felix is an open source implementation of the OSGi Core Release 6 framework specification. -->
```

Comment:
Felix 7.0.1 is a OSGI core 8 implementation, isn't it?

Reply:
According to Wikipedia (could not find anything else) Felix 7 is an OSGI v7 spec implementation. https://en.wikipedia.org/wiki/Apache_Felix

Reply:
Yes, I believe Felix 7.0.1 is an OSGI Core 8 implementation https://en.m.wikipedia.org/wiki/OSGi_Specification_Implementations#Implementations